circuits/crates/libs/polynomial/src/lib.nr

```diff
@@ -117,11 +117,11 @@ impl<let N: u32> Polynomial<N> {
     /// # Arguments
     /// * `upper_bound` - The symmetric bound for coefficient range checking
     ///                   Coefficients must satisfy: -upper_bound <= c <= upper_bound
-    pub fn range_check_1bound<let BIT: u32>(self, upper_bound: u64) {
-        let shifted_bound = (upper_bound as Field) + (upper_bound as Field);
+    pub fn range_check_1bound<let BIT: u32>(self, upper_bound: Field) {
```

Comment:
`range_check_1bound` could just call `range_check_2bounds` with two `upper_bound` parameters at this point no?

Reply:
right!

Reply:
```rust
self.range_check_2bounds::<BIT>(upper_bound, upper_bound);    
```

you mean this @cedoor ?

so we can do like

```rust
self.u.range_check_2bounds::<BIT_U>(bound_params.u_bound, bound_params.u_bound);

// instead of
self.u.range_check_1bound::<BIT_U>(bound_params.u_bound);
```

Reply:
done in [46f29e1](https://github.com/gnosisguild/enclave/pull/972/commits/46f29e15175024422bfe804590c9488f374fd49e)

Reply:
Yess